objects/src/accounts/delta/builder.rs

```diff
@@ -0,0 +1,54 @@
+use alloc::vec::Vec;
+
+use super::{AccountStorageDelta, StorageMapDelta, Word};
+
+#[derive(Clone, Debug, Default)]
+pub struct AccountStorageDeltaBuilder {
+    pub cleared_items: Vec<u8>,
+    pub updated_items: Vec<(u8, Word)>,
+    pub updated_maps: Vec<(u8, StorageMapDelta)>,
+}
+
+impl AccountStorageDeltaBuilder {
+    // CONSTRUCTORS
+    // ===========================================================================================
```

Comment:
minor nit: for section separators inside `impl`'s we usually use single dash line - e.g.,:
```
// CONSTRUCTORS
// --------------------------------------------------------------------------------------------
```

Reply:
changed